.rubocop.yml

```diff
@@ -317,6 +317,9 @@ Style/MultilineMemoization:
 Style/MutableConstant:
   Enabled: false
 
+Style/NumericLiterals:
+  Enabled: false
```

Comment:
Note for future: reasonably sure this is because we're using a numerical literal to represent a date for CURRENT_TOS_VERSION and 20181109 should not in fact be represented as 20_181_109 because it actually represents 2018_11_09.